src/components/Image/__tests__/Image.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  render,
+  fireEvent,
+  getByText,
+  getAllByTestId,
+} from "@testing-library/vue";
+import Image from "./../";
+
+const testImage = "[image location]";
+
+describe("components/Image", () => {
+  it("should have no border class and no 'veil' div when border and opacity properties are not set", () => {
+    const { getByTestId } = render(Image, {
+      props: {
+        src: testImage,
+      },
+    });
+    const image = getByTestId("imageDiv");
+    expect(image.classList.contains("border")).toBe(false);
+    // expect(getAllByTestId(image, "veil").length).toEqual(0); // does not work -> workaround follows:
+    expect(image.innerHTML).toContain("<img"); // make sure we actually have an <img> tag inside our image div
+    expect(image.innerHTML).not.toContain("absolute"); // veil divs have class absolute
```

Comment:
Please check for existence instead of the "absolute" check since this can get flaky.

Reply:
Use this call instead `expect(() => getByTestId("veil")).toThrow();`